### PR TITLE
oc_secret: allow for specifying secret type

### PR DESCRIPTION
**Pull request by jarrpa**
Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

**Comment by jarrpa** [comment]
@abutcher @ashcrow @kwoodson Okay, finally succeeded in smoke testing this and updating it. Review please! :)

File: roles/lib_openshift/library/oc_secret.py [hunk 1]
```diff
@@ -1609,6 +1620,8 @@ def create(self, files=None, contents=None):
 
         secrets = ["%s=%s" % (sfile['name'], sfile['path']) for sfile in files]
         cmd = ['secrets', 'new', self.name]
+        if self.type is not None:
+            cmd.append("--type=%s" % (self.type))
```
Comment (ashcrow) [inline roles/lib_openshift/library/oc_secret.py]: I recommend using ``"".format(..)`` instead of ``%s`` but it's not worth changing right now 😄

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by jarrpa** [comment]
@ashcrow Re-trigger the tests? Looks like a python environment error.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge][severity: blocker]

**Pull request merged**